daemon/mgr/image_utils.go

```diff
@@ -19,7 +20,7 @@ func (mgr *ImageManager) addRegistry(input string) string {
 	if _, ok := reference.Domain(input); ok {
 		return input
 	}
-	return mgr.DefaultRegistry + input
+	return filepath.Join(mgr.DefaultRegistry, mgr.DefaultNamespace, input)
```

Comment:
That means we only specify the image name after merge this pr, just like `pouch pull centos`?

Reply:
no, it just follow original logic, not change it, what I do is to split docker.io/library into docker.io + library.